modules/file-integrity-failure-alerts.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/file-integrity-operator-understanding.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="file-integrity-failure-alerts_{context}"]
+= Prometheus alerts for File Integrity Operator
+
+The File Integrity Operator creates the following default alert in the Operator namespace when a Node has been in a failure state for more than 30 seconds:

```

Comment:
Do we need to add anything that tells use what they need to do in case they see this event?

For example, is there a process we should document here, too?

Reply:
I'm not sure @rhmdnd. I don't think we have a must-gather plug-in for FIO. I pulled this content directly from [the upstream docs](https://github.com/openshift/file-integrity-operator#integrity-failure-alerts).

Reply:
Just curious, is the 30 seconds here is an accurate number or not? Thanks.

Reply:
That's a good point @xiaojiey, the description states 1 second failure.